lib/bech32/bech32.cabal

```diff
@@ -31,7 +31,9 @@ library
   build-depends:
       array
     , base
+    , bifunctors
```

Comment:
Bifunctors are already available in `base`: https://hackage.haskell.org/package/base-4.11.1.0/docs/Data-Bifunctor.html

Reply:
Nicely spotted. I hadn't realized they were available in `base`.